Keep baseline trust values invariant

BendikC opened the pull request:
Changes to implementation so that baselines dont change trust values. Also includes a bit of gameplay for the evaluation

Bendikchristensen committed:
bug fix

fixing inverted trust, right now it skips the message if it trusts the user but we want the opposite

Bendikchristensen committed:
Keeping trust values invariant for baselines

Bendikchristensen committed:
7 game playthroughs with Alice always trust

apostoo commented:
Do we want to push csv's as well?

BendikC commented:
> Do we want to push csv's as well

I'm only pushing the CVs that have the data of my playthroughs. I think it's alright to gather this as it is all the data we will use for evaluation

apostoo reviewed (approved):
Looks good to me then!

Bendikchristensen committed:
Robot waits for 20 seconds post response

Bendikchristensen committed:
Decision making for evaluation

Implemented decision making on every action. For now it is able to handle always trust and never trust properly, but need to finetune for variable and random trust

Bendikchristensen committed:
bug

flipped the decision-making for removing obstacle

Maxjuhh123 committed:
Fixed merge conflicts

Maxjuhh123 committed:
Max TODOs
